observatory-api/observatory/api/server/openapi.yaml.jinja2

```diff
@@ -191,6 +191,7 @@ paths:
           schema:
             $ref: '#/definitions/QueryResponse'
         401:
+          {% if api_client -%}description: API key is missing or invalid{%- endif %}
```

Comment:
Does the api_client not pick up the description from the UnauthorizedError reference defined below?

Reply:
The API client and Cloud Endpoints pick it up fine without the description field, but sphinxcontrib.openapi, which generates the REST API documentation, throws an error if the description field is missing. The OpenAPI specification used in the documentation is generated when api_client is True.

This sphinxcontrib.openapi PR fixes it, however it has been pending since Oct 2021: https://github.com/sphinx-contrib/openapi/pull/120

Reply:
Mmm I see, that's a bit annoying, hopefully it will get updated soon :-)